projects/angular/src/data/datagrid/datagrid-column.ts

```diff
@@ -157,6 +160,18 @@ export class ClrDatagridColumn<T = any>
     this.subscriptions.push(this.listenForDetailPaneChanges());
   }
 
+  get isHidden() {
+    return this.el.nativeElement.classList.contains(HIDDEN_COLUMN_CLASS);
+  }
+
+  get showSeparator() {
+    return this._showSeparator;
+  }
+  set showSeparator(value: boolean) {
+    this._showSeparator = value;
+    this.changeDetectorRef.markForCheck();
+  }
```

Comment:
Would a `BehaviorSubject` and `async` pipe work here instead of manually calling `markForCheck`? I think that would be cleaner and less code.